nf_core/module-template/modules/main.nf

```diff
@@ -26,7 +26,7 @@ process {{ tool_name_underscore|upper }} {
     label '{{ process_label }}'
     publishDir "${params.outdir}",
         mode: params.publish_dir_mode,
-        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:{{ 'meta' if has_meta else "[:]" }}, publish_by_meta:{{ "['id']" if has_meta else "[]" }}) }
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getProcessName(task.process), meta:{{ 'meta' if has_meta else "[:]" }}, publish_by_meta:{{ "['id']" if has_meta else "[]" }}) }
```

Comment:
we probably want `getProcessName(task.process).toLowerCase()` here (or change it in `functions.nf` and have
lowercase process names also in the multiqc report.

Reply:
I think my preference is `getProcessName(task.process).toLowerCase()`, because I would prefer the Process names to be the same case as the Nextflow process in the MultiQC report. 
Are there any strong opinions here?